cmd/ebrelayer/txs/relayToEthereum.go

```diff
@@ -111,12 +106,25 @@ func initRelayConfig(provider string, registry common.Address, event types.Event
 	transactOptsAuth := bind.NewKeyedTransactor(key)
 
 	log.Printf("ethereum tx current nonce from client api is %d\n", nonce)
-	log.Printf("ethereum tx current nonce from local storage is %d\n", nextNonce)
+
+	fmt.Println("suggested gas price: ", gasPrice)
+
+	gasPrice = gasPrice.Mul(gasPrice, big.NewInt(2))
+	fmt.Println("suggested gas price after multiplying by 2: ", gasPrice)
+
+	quarterGasPrice := big.NewInt(0)
+	quarterGasPrice = quarterGasPrice.Div(gasPrice, big.NewInt(4))
+	fmt.Println("quarterGasPrice: ", quarterGasPrice)
```

Comment:
Remove the debug logging

Reply:
Let's merge to sandpit with this, then when we merge to betanet remove it. That way, we can get live debugging while testing it.